taskcluster/scripts/bitrise-schedule.py

```diff
@@ -94,20 +96,26 @@ async def async_main(token, branch, commit, locale):
             await download_log(client, build_slug)
 
 
-async def schedule_build(client, branch, commit, locale):
+async def schedule_build(client, branch, commit, workflow, locale, derived_data_path=None):
     url = BITRISE_URL_TEMPLATE.format(suffix="builds")
+
+    environment_variables = [{
+        "mapped_to": environment_variable_name,
+        "value": environment_variable_value,
+    } for environment_variable_name, environment_variable_value in (
+        ("MOZ_LOCALE", locale),
+        ("MOZ_DERIVED_DATA_PATH", derived_data_path),
```

Comment:
I prefix all variables that aren't part of the regular bitrise flow with `MOZ_`. We use this pattern in mozilla-central too.